src/components/ContactCard/__snapshots__/ContactCard.spec.jsx.snap

```diff
@@ -239,12 +239,12 @@ Array [
 `;
 
 exports[`ContactCard should match snapshot with accounts 1`] = `
-<div>
+Array [
```

Comment:
We're passing from a single node to a node array, I guess it is expected, but why ?

Reply:
This is how enzyme-to-json renders a component that has a fragment root 

https://github.com/adriantoine/enzyme-to-json/pull/90/files#diff-7223bdb24283e1d40122de3c584939c3R136